rtdata/languages/Deutsch

```diff
@@ -1832,9 +1833,9 @@ TP_RAWEXPOS_LINEAR;Korrekturfaktor
 TP_RAWEXPOS_PRESER;Weißpunkt: Lichter schützen
 TP_RAWEXPOS_RGB;Rot, Grün, Blau
 TP_RAWEXPOS_TWOGREEN;Grün-Werte automatisch angleichen
-TP_RAW_1PASSMEDIUM;1-Pass (Medium)
-TP_RAW_3PASSBEST;3-Pass (Beste)
-TP_RAW_4PASS;4-Pass
+TP_RAW_1PASSMEDIUM;1-Pass (Markesteijn)
+TP_RAW_3PASSBEST;3-Pass (Markesteijn)
```

Comment:
Seems the above were reverted to strings from default language file. Is that intentional?
[And I had to google Markesteijn - seems to be an algorithm, is that correct?]

Reply:
Yes, it's all correct. @heckflosse changed it yesterday.